### PR TITLE
oauth2-example-app: clean up oauth server module

### LINKED ISSUE #35: examples: clean up OauthAuthorizationServerModule
The OauthAuthorizationServerModule is a little bit messy as it was part of the previous version of the library. It may be cleaned up for improving the readability of the example app. 

Few TODOs that could be taken as starting point:
 * url params could be removed
 * 1 constructor for the module
 * name to be changed to OAuthAuthorizationServerModule

### PR DESCRIPTION
**Pull request by qnislavnachev**
OauthAuthorizationServerModule get cleaned up
renamed class, removed constructor, url param is removed from
constructors

Fixes #35

### CODE REVIEW

File: oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java [hunk 1]
```diff
@@ -60,67 +60,44 @@ protected OAuth2Config config() {
...
 
   /**
    * constructs the module
    *
-   * @param url
    * @param tokenTimeToLive in seconds
    */
-  public OauthAuthorizationServerModule(String url, Long tokenTimeToLive) {
-    this.url = url;
+  public OAuthAuthorizationServerModule(Long tokenTimeToLive) {
```
Comment (mgenov) [inline oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java]: This constructor should be removed.

File: oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java [hunk 2]
```diff
@@ -60,67 +60,44 @@ protected OAuth2Config config() {
...
     this.tokenTimeToLive = Duration.seconds(tokenTimeToLive);
   }
 
-  public OauthAuthorizationServerModule(String url, String loginPagePath, Long tokenTimeToLive) {
-    this.url = url;
+  public OAuthAuthorizationServerModule(String loginPagePath, Long tokenTimeToLive) {
```
Comment (mgenov) [inline oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java]: Is the loginPath used somewhere?

**Review by mgenov — CHANGES_REQUESTED** [review]

File: oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java (hunk 2 above)
Comment (qnislavnachev) [reply oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java]: in secure module

File: oauth2-example-app/src/main/java/com/clouway/oauth2/app/AppServer.java [hunk 3]
```diff
@@ -33,7 +33,7 @@ public void start() throws Exception {
     context.addEventListener(new GuiceServletContextListener() {
       @Override
       protected Injector getInjector() {
-        return Guice.createInjector(new OauthAuthorizationServerModule("/r/oauth", 60 * 60L), new InMemoryModule());
+        return Guice.createInjector(new OAuthAuthorizationServerModule(60 * 60L), new InMemoryModule());
```
Comment (mgenov) [inline oauth2-example-app/src/main/java/com/clouway/oauth2/app/AppServer.java]: ... loginPage is not configured here.

**Comment by qnislavnachev** [comment]
login path configured and removed unneeded constructor

File: oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java [hunk 4]
```diff
@@ -132,14 +100,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     install(new SitebricksModule() {
       @Override
       protected void configureSitebricks() {
-        at(url + "/login").show(LoginEndpoint.class);
-        at(url + "/userInfo/:token").serve(UserInfoEndPoint.class);
+        at("/r/oauth/login").show(LoginEndpoint.class);
```
Comment (mgenov) [inline oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java:103]: Why this path is "/r/oauth2/login" but the configured one is "/login"?

Comment (qnislavnachev) [reply oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java:103]: Yeah, I check that my bad

Comment (qnislavnachev) [reply oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java:103]: It will be fixed in a minute

Comment (qnislavnachev) [reply oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/OAuthAuthorizationServerModule.java:103]: fixed

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**